src/hipscat_import/verification/arguments.py

```diff
@@ -0,0 +1,38 @@
+"""Utility to hold all arguments required throughout verification pipeline"""
+
+from dataclasses import dataclass, field
+from typing import List, Optional
+
+from hipscat.catalog import Catalog
+
+from hipscat_import.runtime_arguments import RuntimeArguments
+
+
+@dataclass
+class VerificationArguments(RuntimeArguments):
+    """Data class for holding verification arguments"""
+
+    ## Input
+    input_catalog_path: str = ""
+    input_catalog: Optional[Catalog] = None
+
+    ## Verification options
+    field_distribution_cols: List[str] = field(default_factory=list)
```

Comment:
Is there a valid case where this could contain anything other than the RA and dec column names?